docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,85 @@
+# Relative layout
+
+The `RelativeLayout` is a layout class, in which each side of children can be specified as relative to sibling view or parent.
+
+**Relative layout example with `Target` and `RelativeOffset`**
+
+![relativeLayoutExamplePreview](./media/relativeLayoutExamplePreview.png)
```

Comment:
Can we clip the white space of the image? Or you have kept it purposefully?

Reply:
The white space of the image has been removed~